src/Domain/QueryBuilders/QueryBuilderInterface.php

```diff
@@ -8,7 +8,9 @@
 
 interface QueryBuilderInterface
 {
-    public function add(string $type, SyntaxInterface $syntax): void;
```

Comment:
`add` is stuill used, so why is it removed?

Reply:
Yeah, you can add it back if you like. The reason why is because using the `should` / `must` / `filter` methods are less error prone; you cannot provide an unknown type. But I added the `add` method to still support the `addMany` which wasn't in the interface but was in tests.